src/main/java/frc/robot/commands/LaunchHatch.java

```diff
@@ -20,18 +20,25 @@ public LaunchHatch() {
   // Called just before this Command runs the first time
   @Override
   protected void initialize() {
+    Robot.m_hatch.launchHatch();
   }
 
   // Called repeatedly when this Command is scheduled to run
   @Override
   protected void execute() {
-    Robot.m_hatch.launchHatch();
+    if (Robot.m_hatch.isHatchLaunched() == true) {
+      Robot.m_hatch.disableLauncher();
```

Comment:
From Rahul's overall comment, I agree that this should be moved into the "end()" function and "isFinished()" should determine that 0.25 seconds has elapsed from launching the hatch before returning true.